File: MssPayProSaasFront/src/app/modules/group/group/UserGroupDTO.ts
import { SimpleBaseEntity } from "../../../base/base.model";

export class UserGroupDTO {
  id?: number;
  userName?: string;
  firstName?: string;
  lastName?: string;
  emailId?: string;
  password?: string;
  groupName?: string;
  description?: string;
  name?: string;  // Add this line
  tokenGenerated?: boolean; // Add this line
  superUser?: boolean; // Add this property


  roles: any;
  keycloakId: any;
  lastname: string | undefined;
  firstname: string | undefined;
}

File: AdminService/src/main/java/com/mss/adminservice/Config/UserGroupDTO.java
package com.mss.adminservice.Config;

import lombok.Data;

import java.util.UUID;

@Data
public class UserGroupDTO {
    private Long id; // Ensure this is the normal ID from your database
    private String name;
    private String description;
    private String userName; // This should match the JSON field
    private String firstname;
    private String lastName;
    private String emailId;
    private String password;
    private String groupName;
    private Long groupId;

//    public void setGroupId(Long id) {
 //   }
}
